src/components/profile-card.js

```diff
@@ -1,66 +1,101 @@
-    },
-    {
-      id: 3,
-      name: "이수완",
-      age: 22,
-      univ: "이화여자",
-      role: "홍보 파트장",
-      major: "컴퓨터공학",
-      phoneNum: "010-5503-7994",
-      email: "2swan965@gmail.com",
-      githubLink: "https://github.com/lee-soowan65",
-      imageUrl:
-        "https://static.wixstatic.com/media/982853_a188a89325cb4a20b6a487f7e3de5d14~mv2.jpg/v1/crop/x_13,y_1126,w_2998,h_2829/fill/w_351,h_335,al_c,q_80,usm_0.66_1.00_0.01/4A2633DE-3322-4CDC-8F8C-35288D83C43B-500.webp"
-    },
-    {
-      id: 4,
-      name: "최수민",
-      age: 26,
-      univ: "서강",
-      role: "부회장",
-      major: "컴퓨터공학",
-      phoneNum: "010-4413-1261",
-      email: "react@kakao.com",
-      githubLink: "https://github.com/greatSumini",
-      imageUrl:
-        "https://static.wixstatic.com/media/69e8c2_43446e9ac65e4f129083516175259a2b~mv2.jpeg/v1/crop/x_0,y_910,w_3024,h_3122/fill/w_347,h_351,al_c,q_80,usm_0.66_1.00_0.01/%E1%84%8E%E1%85%AC%E1%84%89%E1%85%AE%E1%84%86%E1%85%B5%E1%86%AB.webp"
-    }
-  ];
+export default function ProfileCard (props) {
+  return (
+    <Wrapper>
+    <InfoBox>
+      <InfoMain>
+        <MainTop>
+          <Name_Age>{props.info.name} ({props.info.age})</Name_Age>
+          <Role>{props.info.role}</Role>
+          <Univ_Major>{props.info.univ}대학교 {props.info.major}과</Univ_Major>
+        </MainTop>
+        <MainBottom>
+          <PhoneNum>{props.info.phoneNum}</PhoneNum>
+          <Email> {props.info.email}</Email>
+          <GitLink href={props.info.githubLink}>{props.info.githubLink}</GitLink>
+        </MainBottom>
+      </InfoMain>
+      <Image src={props.info.imageUrl} width="100%"></Image>
+    </InfoBox>
+    <InfoBottom> 신촌 연합 IT 창업 동아리 CEOS</InfoBottom>
+  </Wrapper>
+  );
 
-  return <Wrapper>안녕 나는 프로필 카드</Wrapper>;
 }
 
+//styled-component로 CSS
+
 const Wrapper = styled.div`
-  background-color: red;
-  font-size: 18px;
+    width: 45%;
+    background-color: rgb(255, 255, 255);
+    margin-bottom: 2rem;
+    padding: 1.5rem;
+    border-width: 1px;
+    border-style: solid;
+    border-color: rgb(204, 204, 204);
+    border-image: initial;
+    border-radius: 1.5rem;
+`;
+
+const InfoBox = styled.div`
+    display: flex;
+    flex-direction: row;
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+    margin-bottom: 1.5rem;
+`;
+
+const InfoMain = styled.div`
+    display: flex;
+    flex-direction: column;
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+    width: 50%;
+`;
+
+const InfoBottom = styled.div`
+    display: flex;
+    flex-direction: column;
+    -webkit-box-align: center;
+    align-items: center;
+    font-size: 1.6rem;
+`;
+
+const MainTop = styled.div`
+    display: block;
+`;
+const MainBottom = styled.div`
+    display: block;
 `;
+
+const Name_Age = styled.p`
+    font-size: 2rem;
+    font-weight: bold;
+    margin-bottom: 1rem;
+`;
+const Role = styled.div`
```

Comment:
여기도 p 태그로 써주셨으면 좋았을 것 같아요!
```suggestion
const Role = styled.p`
```